server/suite_test.go

```diff
@@ -191,6 +191,10 @@ var setupSUT = func() {
 	// Inject the mock
 	sut.SetStorageImageServer(imageServerMock)
 	sut.SetStorageRuntimeServer(runtimeServerMock)
+
+	gomock.InOrder(
+		cniPluginMock.EXPECT().Status().Return(t.TestError),
+	)
```

Comment:
@haircommander this breaks some unit tests and will make them hang forever in 
https://github.com/cri-o/cri-o/blob/635b2d98dff9cdbbfd85d2448afa1627410659bd/server/sandbox_run_linux.go#L328

Reply:
Fix in https://github.com/cri-o/cri-o/pull/5099